### PR TITLE
build-configs: android updates

### PR DESCRIPTION
**Pull request by khilman**
Todd Kjos <tkjos@google.com> writes:

> ... please undo my previous request to enable testing on "-r"
> branches
>
> Specifically:
>
> repo: https://android.googlesource.com/kernel/common
>
> 1) android-4.14 will live on and android-4.14-r won't
>
> delete: android-4.14-r
> add: android-4.14
>
> 2) android-4.19 will live on and android-4.19-r won't
>
> delete: android-4.19-r
> add: android-4.19

Signed-off-by: Kevin Hilman <khilman@baylibre.com>

### CODE REVIEW

File: build-configs.yaml [hunk 1]
```diff
@@ -423,7 +423,7 @@ build_configs:
 
   android_4.14-r:
```
Comment (gctucker) [inline build-configs.yaml]: Please update the config name accordingly without `-r` to follow the convention.

Comment (khilman) [reply build-configs.yaml]: ok, done.

Comment (gctucker) [reply build-configs.yaml]: thanks

**Comment by gctucker** [comment]
Tested OK with `kci_build list_configs`.

**Pull request merged**